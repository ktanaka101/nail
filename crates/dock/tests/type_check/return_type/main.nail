fn main() -> string {
  return 10;
}

//---stdout
//---stderr

Error: 
   ╭─[/workspaces/nail/crates/dock/tests/type_check/return_type/main.nail:2:3]
   │
 2 │   return 10;
   │   ────┬────  
   │       ╰────── expected string, actual: int
───╯
