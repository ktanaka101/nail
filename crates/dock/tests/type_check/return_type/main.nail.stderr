[31mError:[0m 
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0m/workspaces/nail/crates/dock/tests/type_check/return_type/main.nail:2:3[38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m2 â”‚[0m [38;5;249m [0m[38;5;249m [0mreturn 10[38;5;249m;[0m
 [38;5;240m  â”‚[0m   â”€â”€â”€â”€â”¬â”€â”€â”€â”€  
 [38;5;240m  â”‚[0m       â•°â”€â”€â”€â”€â”€â”€ expected string, actual: int
[38;5;246mâ”€â”€â”€â•¯[0m
